src/main/resources/runner/docker/haskell/haskell_entry.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# # Set virtual memory limit (Should be big) (in kbytes)
+# ulimit -v $PF_VIR_MEM
+# ulimit -Hv $PF_VIR_MEM
+
```

Comment:
These need to be included, otherwise programs can consume as many resources as they want.

Reply:
I agree. But I still need to figure out the minimum to give them. As the PR says, the stack command says it needs at least 70 MiB.

I can try playing with the numbers asap and push something.

Reply:
Looking at the conversion I think it's like 1 MiB to 1050 kbytes. So you'll need at least 73400, which is less than I have here I think.

Reply:
We may need separate environment files for each language if they're drastically different.

Reply:
I think we can definitely do that.

And I thought it was in bytes, not in kbytes... So, I definitely made it way bigger.

I will change that now and push it. But I can't do anything major this late.

Reply:
BTW, I couldn't find any documentation on the `-u` option for `ulimit`. What is it? Because it was giving me an error saying that it is not supported...

Reply:
The `-u` flag limits the number of user processes that can run and stops the program from spawning too many additional sub-processes (I think like a fork bomb). https://linuxhint.com/linux_ulimit_command/.
It's not supported on all operating systems, if it breaks locally just comment it while testing locally.

Reply:
It doesn't work in the container, says it is not recognized. I am not sure if this something local to me or for every Haskell container.

And I have set the `ulimit` values, check commit [f34b026](https://github.com/puffproject/test-runner/pull/20/commits/f34b026c7cdf7d3283c2086cdcad7d4dd81fa4fd)

Reply:
Works for me, if the values work for you then 👍